phac-nml/irida-next
# Delete Multiple Attachments

## What does this PR do and why?
This PR adds the feature where users can use the checkboxes on the sample attachments page, and then click the `Delete Files` button in order to delete multiple attachments at once.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/82407232/62b3b278-ca07-4d60-a869-00bb79250b3c)
![image](https://github.com/phac-nml/irida-next/assets/82407232/983ef4b2-1aa8-4c8e-848e-82459c2ee40c)
![image](https://github.com/phac-nml/irida-next/assets/82407232/2e0c94cb-3d61-40f7-bff2-654e73bbaabc)

## How to set up and validate locally
1. Navigate to a sample page.
2. Upload attachments
3. Use the checkboxes and selected `>=2` attachments.
4. Click the `Delete Files` button
5. Ensure the list of attachments shown match those selected on the samples attachment page.
6. Delete the attachments
7. Ensure the samples attachment table no longer includes the deleted attachments

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
